admin_guide/manage_authorization_policy.adoc

```diff
@@ -605,56 +786,53 @@ To create a new role from scratch, save this snippet into the file
 
 .Example Role Named exampleview
 ----
-apiVersion: v1
+apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: Role
 metadata:
   name: exampleview
 rules:
-- apiGroups: null
-  attributeRestrictions: null
+- apiGroups:
+  - ""
   resources:
-  - pods
   - builds
+  - pods
   verbs:
   - get
-	- list
-	- watch
+  - list
+  - watch
 ----
 
-Then, to use the current project, run:
+Then, to create the local role, run:
 
 ----
-$ oc project <project_you_want_to_add_the_local_role_exampleview_to>
+$ oc create -f role_exampleview.yaml -n <project_you_want_to_add_the_local_role_exampleview_to>
 ----
 
 Optionally, annotate it with a description.
 
 To use the new role, run:
 
 ----
-$ oc adm policy add-role-to-user exampleview user2
+$ oc adm policy add-role-to-user exampleview user2 --role-namespace=<name_of_project> -n <name_of_project>
```

Comment:
@simo5 I left the `oc adm policy add-*` commands as-is since we have not deprecated any of them.  If we ever do deprecate them, then all this stuff will have to be updated to use the RBAC commands (which are all simple and explicit, but lack a lot of the features of the origin commands).